### DISCUSSION TITLE
Add User Quote Submission

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1066] [Ideas] [answered]
To grow the library of quotes monkeytype has, can you please add an option to allow users to submit quotes. One issue you might face is how would you review the quotes to check if they were inappropriate ?

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I think this is already done through the discord website. In an update a while back called the "quote update" @Miodec said that you could add quotes through the discord so I figure they still do that to expand the quotes.

**Reply by Miodec** [reply]
Yeap, quote submissions are available on the Discord server, in the #quote-submission channel.
